### DISCUSSION TITLE
Is it possible to reference tag by it's name?

### DISCUSSION THREAD

**Discussion by mateusz-glowinski** [gijzelaerr/python-snap7#403] [Q&A] [answered]
Hi,
Is that possible to reference a tag by it's name and not by memory address?
Based on [this](https://github.com/gijzelaerr/python-snap7/discussions/400) topic, I assume that S7-1200 and S7-1500 do store a tags names inside memory.

**Comment by nikteliy** [comment, marked as answer]
Most likely, this is a feature of the extended version of the `S7comm` protocol - `S7comm_plus`. Python-snap7 doesn't support `s7comm_plus`.

**Reply by mateusz-glowinski** [reply]
Ok, thanks.

**Comment by gijzelaerr** [comment]
Closing as answered. Symbolic tag access requires the S7comm+ protocol, which is not the protocol used by this library's classic S7 communication.
